### ISSUE TITLE
[@nativescript/contacts] ReferenceError: CNContactStore is not defined

### ISSUE THREAD

**Issue by lucianomurr** [NativeScript/plugins#225]
### Environment:
Using Typescript.

### Simulator 
on Pixel 5 APIs 30 - Android 11

## package.json
```{
  "name": "nativescript-contacts-demo-ts",
  "main": "app/app.ts",
  "version": "1.0.0",
  "private": true,
  "dependencies": {
    "@nativescript/camera": "^5.0.10",
    "@nativescript/contacts": "^2.0.0",
    "@nativescript/core": "~8.1.1",
    "@nativescript/theme": "~3.0.1",
    "nativescript-permissions": "^1.3.11"
  },
  "devDependencies": {
    "@nativescript/android": "8.1.1",
    "@nativescript/types": "~8.1.1",
    "@nativescript/webpack": "~5.0.0",
    "typescript": "~4.3.5"
  }
}
```

When i try to use this simple call to load contacts
```
getAllContacts(fields?) {
		Contacts.getAllContacts(fields).then(
			(result) => {
				this._updateResult(result.data);
			},
			(err) => {
				console.log(err);
			}
		);
	}
```

I get this error:
```
JS: {
JS:   "message": "Uncaught ReferenceError: CNContactStore is not defined",
JS:   "stackTrace": "ReferenceError: CNContactStore is not defined\n    at self.onmessage (file: app/webpack:/nativescript-demo-ts/node_modules/@nativescript/contacts/worker-get-all-contacts.js:47:0)",
JS:   "filename": "file:///data/data/org.nativescript.nativescriptdemots/files/app/vendor.js",
JS:   "lineno": 1322
JS: }
```

`worker-get-all-contacts.js:47:0`is the iOS version right?
But why in android OS the plugin load the iOS version?

Thanks for the support.

**Comment by NathanWalker** [comment]
@lucianomurr will check shortly, it uses the worker so it may be loading from wrong context.

**Comment by NathanWalker** [comment]
@lucianomurr this is resolved in 2.0.1 now - lemme know if see anything else.
